.github/workflows/dependency-submission.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   dependency-submission:
     runs-on: windows-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
We need to add the hash to this as @bingenito mentioned. @kruplm , any chance you can do that?